packages/datetime/src/dateRangePicker.tsx

```diff
@@ -197,13 +203,22 @@ export class DateRangePicker extends AbstractComponent<IDateRangePickerProps, ID
 
     public render() {
         const modifiers = combineModifiers(this.modifiers, this.props.modifiers);
-        const { className, contiguousCalendarMonths, locale, localeUtils, maxDate, minDate } = this.props;
+        const {
+            className,
+            contiguousCalendarMonths,
+            dayPickerBaseProps,
+            locale,
+            localeUtils,
+            maxDate,
+            minDate,
+        } = this.props;
         const isShowingOneMonth = DateUtils.areSameMonth(this.props.minDate, this.props.maxDate);
 
         const { leftView, rightView } = this.state;
         const disabledDays = [{ before: this.props.minDate }, { after: this.props.maxDate }];
 
-        const dayPickerBaseProps: DayPicker.Props = {
+        const dayPickerProps: ReactDayPicker.Props = {
+            ...dayPickerBaseProps,
```

Comment:
Would need to change this to `...this.props.dayPickerProps` to avoid a naming conflict, or (better approach) we can rename when destructuring above:

```tsx
const {
    // ...
    contiguousCalendarMonths,
    dayPickerProps: dayPickerBaseProps,
    locale,
    // ...
} = this.props;
```